modules/imgimport/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 include_directories(include ../core/include)
+include_directories(include ../core/src)
+include_directories(include ./include)
```

Comment:
If I do not include frame.cpp in headers, it will display error 'undefined reference to Frame::Frame(...)'.